tests/chainerx_tests/unit_tests/routines_tests/test_sorting.py

```diff
@@ -94,3 +122,31 @@ def forward_xp(self, inputs, xp):
         else:
             b = a.argmin(axis)
         return b,
+
+
+# Add cuda tests after cuda implementation of cumsum is supported.
+@op_utils.op_test(['native:0'])
```

Comment:
Why not add cuda tests? The implementation of `CountNonzero` does not depend on `Cumsum`.

Reply:
Yes. I had initially implemented it with `cumsum`. Forgot to modify after the change in implementation. Will add tests accordingly.